src/Compiler/Microsoft.CodeAnalysis.Razor.Compiler/src/Language/RazorConfiguration.cs

```diff
@@ -39,4 +47,24 @@ public override int GetHashCode()
         hash.Add(LanguageServerFlags);
         return hash;
     }
+
+    internal void CalculateChecksum(Checksum.Builder builder)
```

Comment:
Guessing `LanguageServerFlags` can't change ever? Perhaps a comment in the method

Reply:
Added to checksum